sklearn/preprocessing/_encoders.py

```diff
@@ -158,6 +160,19 @@ class OneHotEncoder(_BaseEncoder):
 
         The used categories can be found in the ``categories_`` attribute.
 
+    drop : 'first' or a list/array of shape (n_features,), default=None.
+        Specifies a methodology to use to drop one of the categories per
+        feature. This is useful in situations where perfectly collinear
+        features cause problems, such as when feeding the resulting data
+        into a neural network or an unregularized regression.
+
+        - None : retain all features.
+        - 'first' : drop the first category in each feature. If only one
+                    category is present, the feature will be dropped
```

Comment:
rst/sphinx syntax specificity: when having a list item as this, the full text needs to be aligned with the first word, so something like:

```
- 'first' : drop the first category in each feature. If only one
  category is present, the feature will be dropped ...
```

Otherwise it will be (wrongly) formatted as a definition list, see eg https://46847-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.preprocessing.OneHotEncoder.html